tests/BuildScriptGenerator.Tests/DotnetCore/GlobalJsonSdkResolverTest.cs

```diff
@@ -14,217 +11,106 @@ namespace Microsoft.Oryx.BuildScriptGenerator.Tests.DotNetCore
 {
     public class GlobalJsonSdkResolverTest
     {
-        [Fact]
-        public void NewGlobalJsonModel_SetsAllowPrereleaseProperty_ToTrueByDefault()
-        {
-            // Arrange & Act
-            var model = new GlobalJsonModel();
-            model.Sdk = new SdkModel();
-
-            // Assert
-            Assert.True(model.Sdk.AllowPreRelease);
-        }
-
-        [Fact]
-        public void NewSdk_SetsRollForwardPolicy_ToLatestPatchByDefault()
-        {
-            // Arrange & Act
-            var model = new GlobalJsonModel();
-            model.Sdk = new SdkModel();
-
-            // Assert
-            Assert.Equal(RollForwardPolicy.LatestPatch, model.Sdk.RollForward);
-        }
-
-        [Theory]
-        [InlineData("3.1")]
-        [InlineData("3.1.*")]
-        [InlineData("3.1.102.23")]
-        [InlineData("3.1.102.preview1")]
-        [InlineData("3.1.102-pre1")]
-        public void SdkVersionInfo_TryParseReturnsFalse_ForInvalidVersionFormat(string version)
-        {
-            // Arrange & Act
-            var isValid = SdkVersionInfo.TryParse(version, out var result);
```

Comment:
FYI: The deleted tests here have been moved to a new test file `SdkVersionInfoTest.cs`